homeassistant/components/pilight/base_class.py

```diff
@@ -0,0 +1,186 @@
+"""Base class for pilight."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components import pilight
```

Comment:
Only import the used constants or modules

Reply:
Could you explain further what you mean exactly? I think the mentioned imports are used here.

Reply:
Sure, instead of importing the entire pilight component you could only import the used constants e.g. DOMAIN, SERVICE_NAME and EVENT

Reply:
Thank you for your explanation: I changed it accordingly :)